snapcraft/internal/states/_prime_state.py

```diff
@@ -19,15 +19,15 @@
 from snapcraft.internal.states._state import State
 
 
-def _strip_state_constructor(loader, node):
+def _prime_state_constructor(loader, node):
     parameters = loader.construct_mapping(node)
-    return StripState(**parameters)
+    return PrimeState(**parameters)
 
-yaml.add_constructor(u'!StripState', _strip_state_constructor)
+yaml.add_constructor(u'!PrimeState', _prime_state_constructor)
 
 
-class StripState(State):
-    yaml_tag = u'!StripState'
+class PrimeState(State):
+    yaml_tag = u'!PrimeState'
```

Comment:
This will of course not work on a tree from an old snapcraft (it'll cause the "missing state" error to be raised), but I assume you know that.